man/man4/zfs.4

```diff
@@ -680,6 +680,22 @@ Minimum time "prescient prefetched" blocks are locked in the ARC.
 These blocks are meant to be prefetched fairly aggressively ahead of
 the code that may use them.
 .
+.It Sy zfs_arc_prune_delay_ms Ns = Ns Sy 2000 Ns ms Po 2s Pc Pq ulong
+If nothing is able to be pruned from a superblock, delay our next attempt at
+pruning this superblock for some milliseconds to avoid continually trying to
+do something that can't be done.
+Under pressure, this can save thousands of scans a second per superblock.
+The delay time isn't critical, it needs to be long enough to avoid
+excessive useless work, and short enough that skipping the prune on this
+particular superblock isn't going to cause problems.
+.Pp
+The parameter can be set to 0 (zero) to disable the delay,
```

Comment:
Does this mean that 0 delay = 0 dlay? If so – that's obvious, and it's more confusing to leave it in.

Reply:
Yes. The text is copied from `zfs_arc_shrinker_limit`. It may be redundant but I'm not convinced it's more confusing to have it in: it makes explicit what zero means and contrasts with other params where zero sets some default behavior.